test.sh

```diff
@@ -58,6 +58,23 @@ cd $REPO_ROOT_DIR
 
 # Get a CCM cluster if not already configured (see available settings in dcos-commons/tools/README.md):
 if [ -z "$CLUSTER_URL" ]; then
+    if [ "$SECURITY" = "strict" ]; then
```

Comment:
@jsrodman:  Can you please file a JIRA capturing what refactors you think are necessary for better installation of the DC/OS CLI, so we don't lose track of it.  I agree however that getting smoke tests passing again is the higher priority at the moment as other teams depend on it.  We can refactor from a working state.